crates/router/src/core/payment_methods/cards.rs

```diff
@@ -1184,6 +1193,20 @@ fn filter_pm_based_on_config<'a>(
         .unwrap_or(true)
 }
 
+///Filters the payment method list on basis of Capture methods, checks whether the connector issues Manual payments using cards or not if not it won't be visible in payment methods list
+fn filter_pm_based_on_capture_method_used(
+    payment_method_filters: &settings::PaymentMethodFilters,
+    payment_method_type: &api_enums::PaymentMethodType,
+) -> bool {
+    payment_method_filters
+        .0
+        .get(&settings::PaymentMethodFilterKey::PaymentMethodType(
+            *payment_method_type,
+        ))
+        .and_then(|v| v.absent_impl)
+        .is_none()
```

Comment:
It seems you are not using the value that, was specified for `absent_impl`, even though currently we don't require it, please add a match case, so in the future if any new filters were to be added, the developer would understand the logic that you have written and also the modifications would be much easier